code/dqc/timediff.q

```diff
@@ -0,0 +1,12 @@
+\d .dqc
+
+/ Takes a table name tn and two column names ca and cb, as well as a percentage
+/ pt and a tolerance tl in milliseconds.
+timediff:{[tn;ca;cb;pt;tl]
+  .lg.o[`timediff;"HELLO"];
+  .lg.o[`timediff;"Checking the time differences in columns ",(", "sv string(ca;cb))," of table ",(string tn)];
+  $[tl>re:1-(sum a)%count a:?[tn;();();(>;(0D+tl*00:00:00.001);(-;ca;cb))];
+    (1b;"No major problem with data flow");
```

Comment:
Adding logging for these output messages too.

Reply:
As in.. save the result to a variable, log it, then return  the variable?